client/src/search.tsx

```diff
@@ -345,7 +318,13 @@ function InnerSearchNavigateWidget(props: InnerSearchNavigateWidgetProps) {
               resultItems.length === 0 &&
               inputValue &&
               inputValue !== "/" &&
-              searchIndex && <div className="nothing-found">nothing found</div>
+              searchIndex && (
+                <div className="nothing-found">
+                  No quick results found. Click{" "}
+                  <Link to={searchPath}>here</Link> or press enter to do a
+                  full-search.
```

Comment:
I don't think we need to call it "quick results". Perhaps more accurate, if we need it at all, is; "No document titles found"
<img width="423" alt="Screen Shot 2021-05-24 at 7 24 56 AM" src="https://user-images.githubusercontent.com/26739/119340917-272c5b00-bc61-11eb-885d-8cb893d5f577.png">

Or, just turn the whole thing into:
```javascript
<Link to={searchPath}>Site search for <code>{inputValue}</code></Link>
```